Containerfile.bpfman-agent

```diff
@@ -12,26 +12,18 @@ RUN echo "TARGETOS=${TARGETOS}  TARGETARCH=${TARGETARCH}  BUILDPLATFORM=${BUILDP
 
 WORKDIR /usr/src/bpfman-operator
 
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-
-COPY vendor/ vendor/
-
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
-
-# Copy the go source
+# Copy everything except what's excluded by the .dockerignore file.
 COPY . .
 
+# Verify vendor dependencies are up-to-date.
+RUN ./hack/verify-deps.sh
```

Comment:
do u need to run this here or is it better to run in GH WF for pull request ?

Reply:
Running per commit would be the better option. I can take this out.